src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/AutoCrafterListener.java

```diff
@@ -2,13 +2,19 @@
 
 import java.util.Optional;
 
+import javax.annotation.Nonnull;
 import javax.annotation.ParametersAreNonnullByDefault;
 
+import org.bukkit.Bukkit;
+import org.bukkit.GameRule;
+import org.bukkit.Keyed;
+import org.bukkit.NamespacedKey;
```

Comment:
Is this import for `NamespacedKey` even needed?
Any unused imports should be removed

Reply:
Welp. I just found out I have a bunch of bad imports. My IDE just help me take care of them and I never checked. I probably should check every time I make some changes.